src/HotChocolate/Core/src/Types/Types/Relay/Serialization/OptimizedNodeIdSerializer.cs

```diff
@@ -223,14 +216,7 @@ public unsafe NodeId Parse(string formattedId, Type runtimeType)
             throw SerializerMissing(typeName, rentedBuffer);
         }
 
-        var valueSpan = span.Slice(index + 1);
-        if (valueSpan.IsEmpty)
-        {
-            Clear(rentedBuffer);
-            throw new NodeIdInvalidFormatException(formattedId);
-        }
-
-        var value = ParseValue(valueSerializer, serializer.TypeName, valueSpan);
+        var value = ParseValue(valueSerializer, serializer.TypeName, span.Slice(index + 1));
```

Comment:
@glen-84 this was missing ... it has two parse methods.

Reply:
![D'Oh](https://i.pinimg.com/originals/31/b5/4b/31b54bf1fcbc6ab6b24b185afc390491.jpg)